src/main/java/bio/terra/service/profile/ProfileDao.java

```diff
@@ -41,6 +41,8 @@
         + " FROM billing_profile"
         + " WHERE id in (:idlist)";
 
+    private static final String sqlListOld = "SELECT " + sqlSelectList
```

Comment:
I'm not wild about using terms like "old" and "new" in code that supports versioning. It would be much better if there was a name (or number) for each version to make it obvious which version the code is referring to. All it takes it more than one overlapping change and you've got two different meanings of "old" in the code base.